src/basic/renderers/choice.ts

```diff
@@ -78,6 +69,16 @@ class ChoiceCallbackRenderer implements DestroyableCallbackRenderer, FocusableCa
     });
     formGroup.appendChild(this.input);
 
+    // Add the prompt
+    const prompt = this.callback.getPrompt();
+    if (prompt) {
+      formGroup.classList.add('form-floating');
+      const label = el<HTMLLabelElement>('label');
+      label.innerHTML = prompt;
+      label.htmlFor = id;
+      formGroup.appendChild(label);
+    }
+
```

Comment:
This was moved down from above, the only new code was the `formGroup.classList.add()` for adding the appropriate CSS class.